stratum/hal/lib/tdi/tdi_sde_interface.h

```diff
@@ -153,6 +153,27 @@ class TdiSdeInterface {
 
     // Resets all data fields.
     virtual ::util::Status Reset(int action_id) = 0;
+
+    // Convenience function to update the packet mod meter config in the table data.
+    // This hides the IDs for the $METER_SPEC_* fields.
+    virtual ::util::Status SetPktModMeterConfig(bool in_pps,
```

Comment:
This many discrete parameters to a method is a Bad Code Smell (Long Parameter List). 
The canonical refactoring (Fowler et al.) is to introduce a Parameter Object.

Might I suggest something along the lines of:

```c
  struct PktModMeterConfig {
    bool in_pps,
    uint64 cir_unit;
    uint64 cburst_unit;
    uint64 pir_unit;
    uint64 pburst_unit;
    uint64 cir;
    uint64 cburst;
    uint64 pir;
    uint64 pburst;
    uint64 greenBytes;
    uint64 greenPackets;
    uint64 yellowBytes;
    uint64 yellowPackets;
    uint64 redBytes;
    uint64 redPackets;
  };

  ::util::Status SetPktModMeterConfig(const PktModMeterConfig& config) override;
  ::util::Status GetPktModMeterConfig(PktModMeterConfig& config) override;
```

Reply:
Done. Added struct.h file and followed as mentioned in above comments

Reply:
run the "clang-format -i *.c^C*.h"  before pushing the commit . hope so we should not get indentation issue now